### PR TITLE
Use Fb Sdk in the client side

### LINKED ISSUE #96: Fetch Facebook event data at runtime to prevent builds without a FACEBOOK_GRAPH_TOKEN from failing
Currently, we need to set a valid `FACEBOOK_GRAPH_TOKEN` for the project to build properly. This makes contributing difficult as we can't hand the token out to just anybody.

Perhaps we can at least prevent the builds from failing by just requesting the Facebook event data at runtime using `fetch` instead of through a build-time Gatsby query.

### PR DESCRIPTION
**Pull request by franreysaycon**
**Description**
- strategize a way to use fb sdk in the client side instead at build time.

**Summary of Changes**
- create key hooks such as `useFacebookSdk` and `useFacebookPageEventQuery`

`useFacebookSdk`
- loads the facebook sdk through javascript requests rather than html inclusion to make sure we can tract the loading / error of this module so that there's no react lifecycle problems.

`useFacebookPageEventQuery`
- provides necessary states to keep track of the fb api call.

- created a fb token boundary component to notify devs in order to render the meetup section component. will fail gracefully and not render the component if such env variable does not exist.

- updated `meetupSection.js`

**How to Test**

Test cases
- Run normally without exporting anything. Result should be the meetup section is not rendered as well as key hints at the consoles on what to do.
- Run normally exporting the `FACEBOOK_GRAPH_TOKEN`. Result should show the meetup section.

Closes #96

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -7,69 +7,62 @@ import Container from "./container"
...
         </Flex>
-      </Container>
-    </Box>
-  )
-}
+
+        <ul>
+          {menus.map(menu => (
+            <li key={menu.label}>
+              <Link variant="nav" href={menu.path} fontSize={[1, 2]}>
```
Comment (galacemiguel) [inline src/components/header.js:56]: My bad. 🙈  Thanks for cleaning up after me!

File: src/hooks/useFacebookPageQuery.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+              })
+              return
+            }
+            if (res.error) {
+              dispatch({
+                type: FacebookPageActions.ERROR,
+                payload: res.error,
+              })
+              return
+            }
```
Comment (galacemiguel) [inline src/hooks/useFacebookPageQuery.js:84]: **`NITPICK`**

```suggestion
            if (!res || res.error) {
              dispatch({
                type: FacebookPageActions.ERROR,
                payload: res?.error || { message: "Something went wrong. Empty response." },
              })
              return
            }
```

File: src/hooks/useFacebookPageQuery.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      if (isFbReady) {
+        // eslint-disable-next-line consistent-return
+        // eslint-disable-next-line no-undef
+        FB.api(
+          pageId,
+          { fields: "events", access_token: process.env.FACEBOOK_GRAPH_TOKEN },
+          res => {
+            dispatch({
+              type: FacebookPageActions.PENDING,
+            })
```
Comment (galacemiguel) [inline src/hooks/useFacebookPageQuery.js:70]: Shouldn't we be dispatching pending as early as here? 🤔 

```suggestion
        dispatch({
          type: FacebookPageActions.PENDING,
        })

        // eslint-disable-next-line consistent-return
        // eslint-disable-next-line no-undef
        FB.api(
          pageId,
          { fields: "events", access_token: process.env.FACEBOOK_GRAPH_TOKEN },
          res => {
```

Comment (franreysaycon) [reply src/hooks/useFacebookPageQuery.js:70]: right. haha not used to callbacks like these anymore haha.

File: src/hooks/useFacebookPageQuery.js (hunk 2 above)
Comment (franreysaycon) [reply src/hooks/useFacebookPageQuery.js:84]: gatcha.

**Review by galacemiguel — APPROVED** [review]
Build works without the `FACEBOOK_GRAPH_TOKEN` now, thanks!

**Pull request merged**